modules/apps/frontend-js/frontend-js-translation-web/src/main/resources/META-INF/resources/TranslationModal.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayButtonWithIcon} from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import {ClayInput} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClayManagementToolbar from '@clayui/management-toolbar';
+import ClayModal from '@clayui/modal';
+import ClayTable from '@clayui/table';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const TranslationModal = ({
+	ariaLabels = {
+		default: Liferay.Language.get('default'),
+		manageTranslations: Liferay.Language.get('manage-translations'),
+		translated: Liferay.Language.get('translated'),
+		untranslated: Liferay.Language.get('untranslated'),
+	},
+	defaultLocale,
+	locales,
+	observer,
+	translations,
+}) => {
+	const [active, setActive] = React.useState(false);
+	const [icon, setIcon] = React.useState('search');
+	const inputRef = React.createRef();
+	const [visibleLocales, setVisibleLocales] = React.useState(locales);
+
+	return (
+		<ClayModal observer={observer}>
+			<ClayModal.Header>
+				{Liferay.Language.get('manage-translations')}
+			</ClayModal.Header>
+
+			<ClayModal.Body scrollable>
+				<ClayManagementToolbar aria-label="Management toolbar">
+					<ClayManagementToolbar.Search showMobile={true}>
+						<ClayInput.Group>
+							<ClayInput.GroupItem>
+								<ClayInput
+									aria-label={Liferay.Language.get('search')}
+									className="form-control input-group-inset input-group-inset-after"
```

Comment:
I believe we provide `insetAfter` as a prop on ClayInput

Reply:
Ah Ok, I'll update that (still getting around the API of some components I haven't used much)

Reply:
Updated in [`ab35f14`](https://github.com/liferay-frontend/liferay-portal/pull/1184/commits/ab35f14ab8b71a14ee8cb3edc9873ae8b469fe82)

Reply:
> still getting around the API of some components I haven't used much

no worries! I still have to check the docs 75% because I forget what APIs we offer, even though I wrote a lot of them 😂 

Pretty much any time I find myself having to add a clay class to a clay component, I realize we either already offer the API or we need to add it in. Main exception is for simple util classes

Reply:
> no worries! I still have to check the docs 75% because I forget what APIs we offer, even though I wrote a lot of them :joy: 

I can imagine, there are a bunch of components in there :smile: